samples/nrf9160/cloud_simple/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+CONFIG_LOG=y
+CONFIG_LOG_IMMEDIATE=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_NBIOT=y
```

Comment:
We don't enforce NB-IoT mode in other samples, I'd be consistent here.

Reply:
We probably shouldn't default to NB-IoT anyway, as many operators don't allow TCP traffic

Reply:
Changed to LTE-M for consistency